administrator/components/com_associations/models/forms/filter_associations.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<field
+		name="associationcomponent"
+		type="associatedcomponent"
+		label="COM_ASSOCIATIONS_FILTER_SEARCH_LABEL"
+		description="COM_ASSOCIATIONS_FILTER_SEARCH_DESC"
```

Comment:
all `COM_ASSOCIATIONS_*`  language vars in this file need to be in the language file.

for normalization see how the other componets does it.